Gemfile

```diff
@@ -21,15 +21,13 @@ gem 'turbo-rails'
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 
```

Comment:
- [ ] It would be better if we solve the `n+1` query problem in our post index method, here is one way we can fix it `@posts = @user.posts.includes(:comments)` in this way we are fetching all posts and comments of a user